serverless/verify-backup/scripts/verify-backup.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# Downloads latest backup dump and decrypts it
+npm run decrypt-dump
+
+# Find for existing db; If exists, drop previously created db
+echo 'Preparing db'
+su - postgres -c "psql -lqt | cut -d \| -f 1 | grep -qw postmangovsg_dev && dropdb postmangovsg_dev"
+su - postgres -c "createdb postmangovsg_dev"
+
+# Restore dump; On error, print error message and exit script
+echo 'Restoring data dump'
+pg_restore --no-owner -h localhost -p 5432 -U postgres -d postmangovsg_dev postman.decrypted.dump || \
+{ echo 'Data dump restoration failed!' && exit 1; }
+
+echo 'Making simple query on db'
+su - postgres -c "psql -d postmangovsg_dev -c 'SELECT 1+1'"
```

Comment:
- if the verification query fails, script should exit with an error code. 
- can we notify slack on success? eg with number of users, campaigns, messages etc, for sanity check. Can be done in separate PR.

Reply:
Noted, will create a separate issue for that